app/Http/Controllers/Api/WaitingRoomsController.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Gladiator\Http\Controllers\Api;
+
+use Gladiator\Models\WaitingRoom;
+use Illuminate\Http\Request;
+
+class WaitingRoomsController extends ApiController
+{
+    public function calculateSplit(Request $request)
```

Comment:
Cool :) I think unless it's something that we'd document in our API docs etc and potentially used by external API or whatevs, let's just keep those methods to internal controllers.